fdbserver/Status.actor.cpp

```diff
@@ -364,7 +341,10 @@ static JsonBuilderObject machineStatusFetcher(WorkerEvents mMetrics, vector<Work
 				machineJsonMap[machineId] = statusObj;
 			}
 
-			if (configuration.present() && !configuration.get().isExcludedServer(it->first))
+			//FIXME: this will not catch if the secondary address of the process was excluded
```

Comment:
Is this FIXME intended to be fixed before merge, or are we deferring it until later?

Reply:
Deferred, basically it just means the machine exclusion list in status could be inaccurate, but since this is a pretty minor problem, and a pre-existing bug I am not planning on fixing it.